shared/scss/main.scss

```diff
@@ -40,7 +40,6 @@
 @import "components/ckan/additional-info";
 @import "components/ckan/page-header";
 @import "components/ckan/activity";
-@import "components/ckan/form";
 @import "components/ckan/list-group";
```

Comment:
Styles from components/ckan/form were moved to components/common/form to be shared between ckan and wp